src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './Sidebar.css';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import HomeIcon from '@mui/icons-material/Home';
+import TagIcon from '@mui/icons-material/Tag';
+import NotificationsNoneIcon from '@mui/icons-material/NotificationsNone';
+import MailOutlineIcon from '@mui/icons-material/MailOutline';
+import BookmarkBorderIcon from '@mui/icons-material/BookmarkBorder';
+import ListAltIcon from '@mui/icons-material/ListAlt';
+import Person2Icon from '@mui/icons-material/Person2';
+import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
+import Button from '@mui/material/Button';
+import SidebarOption from './SidebarOption';
+
+function Sidebar() {
+  return (
+    <div className="sidebar">
+      {/* Twitter Icon */}
+      <TwitterIcon />
+      {/* home Icon */}
+      <SidebarOption active Icon={HomeIcon} text="Home" />
+
+      {/* TagIcon */}
+      <SidebarOption Icon={TagIcon} text="Explore" />
+
+      {/* Notificcation Icon */}
+      <SidebarOption Icon={NotificationsNoneIcon} text="Notifications" />
+
+      {/* Messege Icon  */}
+      <SidebarOption Icon={MailOutlineIcon} text="Messeges" />
+
+      {/* Bookmark Icon */}
+      <SidebarOption Icon={BookmarkBorderIcon} text="Bookmarks" />
+
+      {/* List Icon */}
+      <SidebarOption Icon={ListAltIcon} text="Lists" />
+
+      {/* Profile Icon  */}
+      <SidebarOption Icon={Person2Icon} text="Profile" />
+
+      {/* More Icon */}
+      <SidebarOption Icon={MoreHorizIcon} text="More" />
```

Comment:
It would be a good idea to make your code clean and condensed by following DRY  approach. You can create a `sidebarOptionMenus` array object that takes in `Id`, `iconName` and `menuText`as its properties. That way you can iterate through all the menu items and pass it in your `SidebarOption` component.

Something like this in your jsx:
```
let sidebarOptionMenus = [{
   id: 1,
   iconName: {HomeIcon},
   menuText: `Home`
}, 
{
   id: 2,
   iconName: {TagIcon},
   menuText: `Explore`
},
...
]

let sidebarOptionMenusList = sidebarOptionMenus.map((item)=>{
  return <SidebarOption key={item.id} Icon={item.iconName} text={item.menuText} />
}
```

In your html
```
<TwitterIcon />
{sidebarOptionMenusList}
<Button varient="outlined" className="tweet-btn" fullWidth>
```